src/EditorFeatures/Core/Shared/Tagging/Utilities/TagSpanIntervalTree.cs

```diff
@@ -49,21 +53,23 @@ public bool HasSpanThatContains(SnapshotPoint point)
         }
 
         public IList<ITagSpan<TTag>> GetIntersectingSpans(SnapshotSpan snapshotSpan)
+        {
+            var pooledResult = Classifier.GetPooledList<ITagSpan<TTag>>(out var result);
```

Comment:
I assume there are some remaining callers of GetIntersectingSpans that cant be switched to AddIntersectingSpans?

Reply:
yesn't.  THere are callers.  They could be switched to AddXXX.  I'd prefer to do that in a followup to keep this PR under control :)